src/nci_util.c

```diff
@@ -175,6 +175,17 @@ nci_parse_mode_param(
 
             memset(ppb, 0, sizeof(*ppb));
             memcpy(ppb->nfcid0, bytes + 1, 4);
+            memcpy(ppb->app_data, bytes + 5, 4);
+            ppb->prot_info_len = bytes[0] - 8;
+            /* NFCForum-TS-DigitalProtocol-1.0
+             * Page 61: SFGI has a value in the range from 1 to 14. The value
+             * 15 is RFU.
+             *
+             * Set default value of optional byte to RFU value to indicate
+             * that it's abscent.
+             */
```

Comment:
This comment is unnecessary, SFGI is already mentioned in the comment above. Since SFGI is not being analyzed or used by this code, it's enough to just mention that it's there (and it's already mentioned earlier). Whoever wants to know what it means, should read the spec.